dev/Dockerfile

```diff
@@ -0,0 +1,28 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+FROM thrift:0.12
+
+RUN set -ex; \
+  apt-get update; \
+  apt-get install -y \
```

Comment:
It is recommended to add `--no-install-recommends`

Reply:
curious that's contra intuitive, I would expect to eventually need the 'recommended' packages save if I am obsessed to have a small image.
Done.

Reply:
Thanks. In practice, the difference in size is minimal, but it won't install stuff that you're not using.